docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -219,6 +219,31 @@ _sql_db_isclean(){
     esac
 }
 
+_couch_db_isclean(){
+  # assume that we can connect to local CouchDB
+  # assume that env contains local couch user and password.
+  echo "$FUNCNAME: Checking if local couch db is empty"
+  agent_summary=$(curl -s -u "$COUCH_USER:$COUCH_PASS" http://$COUCH_HOST:$COUCH_PORT/wmagent_summary)
+
+  if ! type jq > /dev/null ; then
+    echo "$FUNCNAME: jq is not installed, we can not check if couchdb is empty"
+    return $(false)
```

Comment:
To increase the compatibility with the current images, I would suggest to simply skip this check if `jq` is not available (instead of failing the check).

Reply:
ok, i agree, however keep in mind that this script is copied [1] inside the docker image, so nothing is going to change for existing images

---

[1] https://github.com/dmwm/CMSKubernetes/blob/e1106b4aa00fef9b1a655c6505ea20a8d7e5cbe1/docker/pypi/wmagent/Dockerfile#L41